app/src/commander.cpp

```diff
@@ -57,7 +57,37 @@ void CmdHandler::handle(CmdSetUsbChannelMsg& msg)
         }
     }
 }
-
+//------------------------------------------------------------------------------
+void CmdHandler::handle(CmdGetPwrRelayMsg& msg)

```

Comment:
We don't need this as a comamnd, just add another row to inf command

Reply:
Of course, is a part of pwr command right now.